### ISSUE TITLE
Build for TF 1.2.1

### ISSUE THREAD

**Issue by cancan101** [rainbean/TensorflowPod#2]
Tensorflow 1.2.1 has been released: https://github.com/tensorflow/tensorflow/releases/tag/v1.2.1

**Comment by rainbean** [comment]
1.2.1 is quite minor update, the only fix is to rollback limit of dropout rate for Python training layer. No code change in iOS (C++) part. So literally it equal to 1.2.0. 

Still looking forward to 1.3.0 release, which should be in the corner soon.
